docs/src/components/ExternalExampleLayout.tsx

```diff
@@ -17,11 +20,26 @@ const ExternalExampleLayout: any = props => {
   }, examplePaths)
 
   if (!examplePath) return <PageNotFound />
-
-  const ExampleComponent = exampleContext(examplePath).default
-  if (!ExampleComponent) return <PageNotFound />
-
-  return <ExampleComponent />
+  const exampleSource: ExampleSource = exampleSourcesContext(examplePath)
+
+  return (
+    <SourceRender
+      babelConfig={babelConfig}
+      source={exampleSource.js}
+      renderHtml={false}
+      resolver={importResolver}
+    >
+      <SourceRender.Consumer>
+        {({ element, error }) => (
+          <>
+            {element}
+            {/* This block allows to see issues with examples as visual regressions. */}
+            {error && <div style={{ fontSize: '5rem', color: 'red' }}>{error.toString()}</div>}
```

Comment:
It's a little clumsy, but it will show us broken examples as visual regressions.